src/resources/create/extensions/format-typst/_extensions/qstart-filesafename-qend/typst-template.typ

```diff
@@ -68,15 +72,21 @@
 
   if abstract != none {
     block(inset: 2em)[
-    #text(weight: "semibold")[Abstract] #h(1em) #abstract
+    #text(weight: "semibold")[$labels.abstract$] #h(1em) #abstract
```

Comment:
Just a note that I don't think we explain anywhere what variables in template can be used for internationalization support. 

By replacing a whole template like this, it means a user would need to now what to use to make this happens. 

Maybe we need to open an issue about the need to document this ? It was hidden in our internal template logic previously, but now we expose it explicitly in `quarto create extension` for typst format via this template.  So it is seems more important than before to explain this. 

What do you all think ?